ds_judgements_public_ui/templates/pages/style_guide.html

```diff
@@ -54,436 +54,419 @@ <h3>Checkboxes</h3>
+              <li>third party rights the Licensor is not authorised to license; and</li>
+              <li>information subject to other intellectual property rights, including patents, trademarks, and design rights.</li>
+            </ul>
+            <p>For the avoidance of doubt:</p>
+            <ul>
+              <li>this licence is not a data sharing agreement for personal data; and</li>
+              <li>this licence is not a processing agreement for personal data.</li>
+            </ul>
             <p>
-              Under this licence you can copy, publish, distribute and transmit the case law data and use the data commercially, for example, by combining it with
-              other Information, or by including it in your own product or application
+              Data Controllers will need to satisfy themselves that they have a legal basis and valid purpose for processing
+              personal data.
             </p>
-            <button type="submit" class="govuk-button" data-module="govuk-button">I want to perform computational analysis</button>
           </div>
+        </details>
+        <div class="info-box__content">
+          <h3 class="info-box__heading">I want to perform computational analysis</h3>
+          <p>
+            Under this licence you can copy, publish, distribute and transmit the case law data and use the data commercially, for example, by combining it with
+            other Information, or by including it in your own product or application
+          </p>
+          <button type="submit" class="govuk-button" data-module="govuk-button">I want to perform computational analysis</button>
         </div>
       </div>
-      <h3>Notification banners</h3>
-      <p>To tell the user about something they need to know about, but that’s not directly related to the page content</p>
-      <div class="component-wrapper">
-        <h3>Aqua blue banner</h3>
-        <div class="govuk-notification-banner_aquablue"
-             role="region"
-             aria-labelledby="govuk-notification-banner-title"
-             data-module="govuk-notification-banner">
-          <div class="govuk-notification-banner__header">
-            <h2 class="govuk-notification-banner__title"
-                id="govuk-notification-banner-title">Important</h2>
-          </div>
-          <div class="govuk-notification-banner__content">
-            <p class="govuk-notification-banner__heading">
-              You have 7 days left to send your application.
-              <a class="govuk-notification-banner__link" href="#">View application</a>.
-            </p>
-          </div>
+    </div>
+    <h3>Notification banners</h3>
+    <p>To tell the user about something they need to know about, but that’s not directly related to the page content</p>
+    <div class="component-wrapper">
+      <div class="govuk-notification-banner_red"
+           role="region"
+           aria-labelledby="govuk-notification-banner-title"
```

Comment:
Can I ask for a screenshot on the front of the PR for these forms?

Reply:
context: https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/adr/0015-translations.md